src/app/pages/forms-and-validation/phone-number-format/phone-number-format.component.html

```diff
@@ -0,0 +1,47 @@
+<mat-toolbar color="primary" class="app-bar mat-elevation-z4" data-cy="pxb-toolbar">
+    <button data-cy="toolbar-menu" *ngIf="isSmall" style="margin-left: -8px" mat-icon-button (click)="toggleMenu()">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">Phone Number Format</div>
+</mat-toolbar>
+<section class="mat-typography">
```

Comment:
Why is this class called "mat-typography"? It doesn't seem appropriately descriptive of what this element is.